src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/WithdrawalTest.kt

```diff
@@ -1313,9 +1300,12 @@ class WithdrawalTest : IntegrationTestBase() {
     return application
   }
 
+  data class DatePeriod(val start: LocalDate, val duration: Int)
```

Comment:
could it be confusing that we already have an API class of `DatePeriod` with a different construction `(start_date, end_date)`? Perhaps it's not an issue as this is just a local class for testing? But if you think it could be confusing, then a different class name like `DateSpan` might work?

Reply:
I agree, it can be confusing. I'll change it DateSpan in my next PR and also make it private